front/src/applications/stdcmV2/views/StdcmViewV2.tsx

```diff
@@ -3,18 +3,30 @@ import React, { useState } from 'react';
 import { useSelector } from 'react-redux';
 
 import type { ManageTrainSchedulePathProperties } from 'applications/operationalStudies/types';
+import type { StdcmV2SuccessResponse } from 'applications/stdcm/types';
 import { useOsrdConfSelectors } from 'common/osrdContext';
+import { useStoreDataForSpeedLimitByTagSelector } from 'common/SpeedLimitByTagSelector/useStoreDataForSpeedLimitByTagSelector';
+import { useStoreDataForRollingStockSelector } from 'modules/rollingStock/components/RollingStockSelector/useStoreDataForRollingStockSelector';
+import { getTrainScheduleV2Activated } from 'reducers/user/userSelectors';
 
 import StdcmConfig from '../components/StdcmConfig';
 import StdcmHeader from '../components/StdcmHeader';
+import StdcmResults from '../components/StdcmResults';
 import type { StdcmSimulationResult } from '../types';
 
 const StdcmViewV2 = () => {
   const { getScenarioID } = useOsrdConfSelectors();
   const scenarioID = useSelector(getScenarioID);
 
+  const trainScheduleV2Activated = useSelector(getTrainScheduleV2Activated);
+  const { speedLimitByTag } = useStoreDataForSpeedLimitByTagSelector();
+  const { rollingStock } = useStoreDataForRollingStockSelector();
+
   const [pathProperties, setPathProperties] = useState<ManageTrainSchedulePathProperties>();
 
+  const [stdcmSimulationResults, setStdcmSimulationResults] = useState<StdcmV2SuccessResponse>();
```

Comment:
If you get the results directly here from the hook, then you can remove this state cont:
```suggestion
const {
    launchStdcmRequest,
    cancelStdcmRequest,
    stdcmV2Results,
    pathProperties,
    requestIsPending,
  } = useStdcm();
```

Reply:
fix